queries.sql

```diff
@@ -103,4 +103,75 @@ WHERE owners.full_name ='Dean Winchester' AND animals.escape_attempts = 0;
 
 SELECT owners.full_name as OWNER , COUNT(animals.name) as count FROM
 owners JOIN animals ON owners.id = animals.owners_id GROUP BY owners.full_name
-ORDER BY count DESC LIMIT 1;
\ No newline at end of file
+ORDER BY count DESC LIMIT 1;
+
+/* Who was the last animal seen by William Tatcher? */
+
+SELECT vets.name AS vet, animals.name AS animal, 
+   visits.date_of_visits AS visited_date 
+   FROM visits 
+JOIN vets ON vets_id = vets.id 
+JOIN animals ON animals_id = animals.id 
+WHERE vets_id =
+  (SELECT id FROM vets WHERE name = 'William Tatcher') 
+ORDER BY date_of_visits DESC LIMIT 1;
```

Comment:
- [ ] The correct order is:- 
```
`ORDER BY visited_date DESC LIMIT 1;
```
There should be `visits.visit_date`. Since you have used `visits.visit_date` as `visited_date` on line 111. You can use it `visited_date` directly.